07_deploy_masters.sh

```diff
@@ -48,7 +48,7 @@ done
 # https://storyboard.openstack.org/#!/story/2005093
 NUM_ACTIVE=$(openstack baremetal node list --fields name --fields provision_state | grep master | grep active | wc -l || echo 0)
 while [ "$NUM_ACTIVE" != "3" ]; do
-  if openstack baremetal node list --fields name --fields provision_state | grep master | grep error; then
+  if openstack baremetal node list --fields name --fields provision_state | grep master | grep error | grep failed; then
```

Comment:
I'm not sure this will work, because the grep error will already have filtered entry with "failed" status?

Perhaps something like grep -e error -e failed, or a conditional regex would be better?

Reply:
You're absolutely right (as usual).
Corrected.